assets/quick-add.css

```diff
@@ -88,7 +88,7 @@
 
   .quick-add-modal__content-info--bulk thead {
     position: sticky;
-    z-index: 2;
+    z-index: 3;
```

Comment:
Why this change? Should we make this a sufficiently large value if we want this to appear on the top at all times?

Reply:
Here is why https://share.descript.com/view/wOls6SoXhau